### ISSUE TITLE
./mad deps compile plan repl fails

### ISSUE THREAD

**Issue by laapsaap** [5HT/n2o#212]
```
[admin@nuc5-i3 samples]$ ./mad deps compile plan repl
Deps Params: []
==> dependency: "git://github.com/synrc/mad" tag: {tag,"master"}
==> dependency: "git://github.com/proger/ling" tag: {tag,"osx-again"}
==> dependency: "git://github.com/synrc/sh" tag: {tag,"1.4"}
==> dependency: "git://github.com/evanmiller/erlydtl" tag: {tag,"0.8.0"}
==> dependency: "git://github.com/uwiger/gproc.git" tag: {tag,"0.3"}
==> dependency: "git://github.com/synrc/fs" tag: {tag,"0.9"}
==> dependency: "git://github.com/synrc/active" tag: {tag,"0.9"}
==> dependency: "git://github.com/synrc/rest" tag: {tag,"1.5.0"}
==> dependency: "git://github.com/synrc/kvs" tag: {tag,"2.1"}
==> dependency: "git://github.com/5HT/n2o" tag: {tag,"master"}
==> dependency: "git://github.com/extend/cowboy" tag: {tag,"1.0.1"}
==> dependency: "git://github.com/ninenines/cowlib.git" tag: "1.0.0"
==> dependency: "git://github.com/ninenines/ranch.git" tag: "1.0.0"
Compile Params: []
==> "mad"
==> "ling"
==> "sh"
Compiling /deps/sh/src/sh_sup.erl
Compiling /deps/sh/src/sh_path.erl
Compiling /deps/sh/src/sh_app.erl
Compiling /deps/sh/src/sh.erl
Writing /deps/sh/ebin/sh.app
Compiling /deps/mad/src/mad_utils.erl
Compiling /deps/mad/src/mad_static.erl
Compiling /deps/mad/src/mad_run.erl
Compiling /deps/mad/src/mad_repl.erl
Compiling /deps/mad/src/mad_release.erl
Compiling /deps/mad/src/mad_plan.erl
Compiling /deps/mad/src/mad_ling.erl
/home/admin/Repositories/n2o/samples/deps/mad/src/mad_ling.erl:8: Warning: variable 'App' is unused
/home/admin/Repositories/n2o/samples/deps/mad/src/mad_ling.erl:15: Warning: variable 'B' is unused
/home/admin/Repositories/n2o/samples/deps/mad/src/mad_ling.erl:85: Warning: variable 'App' is unused
Compiling /deps/mad/src/mad_deps.erl
Compiling /deps/mad/src/mad_create.erl
Compiling /deps/mad/src/mad_compile.erl
Compiling /deps/mad/src/mad_bundle.erl
Compiling /deps/mad/src/mad.erl
Compiling /deps/mad/src/ling/ling_lib.erl
Compiling /deps/mad/src/ling/ling_iopvars.erl
Compiling /deps/mad/src/ling/ling_iops.erl
Compiling /deps/mad/src/ling/ling_disasm.erl
Compiling /deps/mad/src/ling/ling_code.erl
Compiling /deps/mad/src/ling/ling_bifs.erl
Compiling /deps/mad/src/ling/bfd_objcopy.erl
Compiling /deps/mad/src/deploy/mad_voxoz.erl
Compiling /deps/mad/src/deploy/mad_heroku.erl
Compiling /deps/mad/src/compile/mad_yecc.erl
Compiling /deps/mad/src/compile/mad_script.erl
Compiling /deps/mad/src/compile/mad_port.erl
Compiling /deps/mad/src/compile/mad_none.erl
Compiling /deps/mad/src/compile/mad_leex.erl
Compiling /deps/mad/src/compile/mad_erl.erl
Compiling /deps/mad/src/compile/mad_dtl.erl
Compiling /deps/mad/src/compile/mad_app.erl
Writing /deps/mad/ebin/mad.app
==> "erlydtl"
Compiling /deps/erlydtl/src/erlydtl_parser.erl
Compiling /deps/erlydtl/src/erlydtl_unparser.erl
Compiling /deps/erlydtl/src/erlydtl_scanner.erl
Compiling /deps/erlydtl/src/erlydtl_runtime.erl
Compiling /deps/erlydtl/src/erlydtl_i18n.erl
Compiling /deps/erlydtl/src/erlydtl_filters.erl
Compiling /deps/erlydtl/src/erlydtl_compiler.erl
Compiling /deps/erlydtl/src/erlydtl.erl
Compiling /deps/erlydtl/src/i18n/sources_parser.erl
Compiling /deps/erlydtl/src/i18n/po_generator.erl
Compiling /deps/erlydtl/src/i18n/i18n_manager.erl
Compiling /deps/erlydtl/src/i18n/blocktrans_scanner.erl
Compiling /deps/erlydtl/src/i18n/blocktrans_parser.erl
Compiling /deps/erlydtl/src/i18n/blocktrans_extractor.erl
Compiling /deps/erlydtl/src/i18n/po_scanner.erl
Compiling /deps/erlydtl/src/filter_lib/erlydtl_dateformat.erl
Compiling /deps/erlydtl/src/filter_lib/erlydtl_slice.erl
Compiling /deps/erlydtl/src/erlydtl_deps.erl
Compiling /deps/erlydtl/src/erlydtl_contrib_humanize.erl
Writing /deps/erlydtl/ebin/erlydtl.app
==> "gproc"
Compiling /deps/gproc/src/gproc_pool.erl
Compiling /deps/gproc/src/gproc_lib.erl
Compiling /deps/gproc/src/gproc_info.erl
Compiling /deps/gproc/src/gproc_dist.erl
/home/admin/Repositories/n2o/samples/deps/gproc/src/gproc_dist.erl:23: Warning: behaviour gen_leader undefined
Compiling /deps/gproc/src/gproc.erl
Compiling /deps/gproc/src/gproc_sup.erl
Compiling /deps/gproc/src/gproc_pt.erl
Compiling /deps/gproc/src/gproc_ps.erl
Compiling /deps/gproc/src/gproc_monitor.erl
Compiling /deps/gproc/src/gproc_init.erl
Compiling /deps/gproc/src/gproc_bcast.erl
Compiling /deps/gproc/src/gproc_app.erl
Writing /deps/gproc/ebin/gproc.app
==> "fs"
Compiling /deps/fs/src/fs_sup.erl
Compiling /deps/fs/src/fs.erl
Compiling /deps/fs/src/sys/inotifywait.erl
Compiling /deps/fs/src/sys/fsevents.erl
Compiling /deps/fs/src/sys/fanotify.erl
Compiling /deps/fs/src/fs_server.erl
Compiling /deps/fs/src/fs_event_bridge.erl
Compiling /deps/fs/src/fs_app.erl
Writing /deps/fs/ebin/fs.app
==> "active"
Compiling /deps/active/src/active_sup.erl
Compiling /deps/active/src/active.erl
/home/admin/Repositories/n2o/samples/deps/active/src/active.erl:43: Warning: variable 'App' is unused
/home/admin/Repositories/n2o/samples/deps/active/src/active.erl:44: Warning: variable 'App' is unused
/home/admin/Repositories/n2o/samples/deps/active/src/active.erl:45: Warning: variable 'App' is unused
/home/admin/Repositories/n2o/samples/deps/active/src/active.erl:46: Warning: variable 'App' is unused
/home/admin/Repositories/n2o/samples/deps/active/src/active.erl:49: Warning: variable 'Else' is unused
/home/admin/Repositories/n2o/samples/deps/active/src/active.erl:60: Warning: variable 'App' is unused
Compiling /deps/active/src/active_app.erl
Writing /deps/active/ebin/active.app
==> "rest"
Compiling /deps/rest/src/rest_cowboy.erl
Compiling /deps/rest/src/rest.erl
Compiling /deps/rest/src/rest_sup.erl
Compiling /deps/rest/src/rest_app.erl
Writing /deps/rest/ebin/rest.app
==> "kvs"
Compiling /deps/kvs/src/store_sql.erl
Compiling /deps/kvs/src/store_riak.erl
/home/admin/Repositories/n2o/samples/deps/kvs/src/store_riak.erl:146: Warning: variable 'Options' is unused
Compiling /deps/kvs/src/store_mnesia.erl
Compiling /deps/kvs/src/store_kai.erl
/home/admin/Repositories/n2o/samples/deps/kvs/src/store_kai.erl:14: Warning: variable 'Node' is unused
/home/admin/Repositories/n2o/samples/deps/kvs/src/store_kai.erl:27: Warning: variable 'Object' is unused
/home/admin/Repositories/n2o/samples/deps/kvs/src/store_kai.erl:27: Warning: variable 'Record' is unused
/home/admin/Repositories/n2o/samples/deps/kvs/src/store_kai.erl:59: Warning: variable 'IndexId' is unused
/home/admin/Repositories/n2o/samples/deps/kvs/src/store_kai.erl:59: Warning: variable 'IndexVal' is unused
/home/admin/Repositories/n2o/samples/deps/kvs/src/store_kai.erl:59: Warning: variable 'Tab' is unused
Compiling /deps/kvs/src/kvs_user.erl
Compiling /deps/kvs/src/kvs_sup.erl
Compiling /deps/kvs/src/kvs_feed.erl
Compiling /deps/kvs/src/kvs_acl.erl
Compiling /deps/kvs/src/kvs.erl
/home/admin/Repositories/n2o/samples/deps/kvs/src/kvs.erl:231: Warning: variable 'Id' is unused
/home/admin/Repositories/n2o/samples/deps/kvs/src/kvs.erl:238: Warning: variable 'Id' is unused
Compiling /deps/kvs/src/kvs_subscription.erl
Compiling /deps/kvs/src/kvs_app.erl
Writing /deps/kvs/ebin/kvs.app
==> "n2o"
==> "cowboy"
==> "cowlib"
Compiling /deps/cowlib/src/cow_spdy.erl
Compiling /deps/cowlib/src/cow_qs.erl
Compiling /deps/cowlib/src/cow_multipart.erl
Compiling /deps/cowlib/src/cow_mimetypes.erl
Compiling /deps/cowlib/src/cow_http_te.erl
Compiling /deps/cowlib/src/cow_http_hd.erl
Compiling /deps/cowlib/src/cow_http.erl
Compiling /deps/cowlib/src/cow_date.erl
Compiling /deps/cowlib/src/cow_cookie.erl
Writing /deps/cowlib/ebin/cowlib.app
==> "ranch"
Compiling /deps/ranch/src/ranch_transport.erl
Compiling /deps/ranch/src/ranch_tcp.erl
Compiling /deps/ranch/src/ranch_ssl.erl
Compiling /deps/ranch/src/ranch_conns_sup.erl
Compiling /deps/ranch/src/ranch_acceptor.erl
Compiling /deps/ranch/src/ranch.erl
Compiling /deps/ranch/src/ranch_sup.erl
Compiling /deps/ranch/src/ranch_server.erl
Compiling /deps/ranch/src/ranch_protocol.erl
Compiling /deps/ranch/src/ranch_listener_sup.erl
Compiling /deps/ranch/src/ranch_app.erl
Compiling /deps/ranch/src/ranch_acceptors_sup.erl
Writing /deps/ranch/ebin/ranch.app
Compiling /deps/cowboy/src/cowboy_websocket_handler.erl
Compiling /deps/cowboy/src/cowboy_websocket.erl
/home/admin/Repositories/n2o/samples/deps/cowboy/src/cowboy_websocket.erl:18: Warning: behaviour cowboy_sub_protocol undefined
Compiling /deps/cowboy/src/cowboy_sub_protocol.erl
Compiling /deps/cowboy/src/cowboy_static.erl
Compiling /deps/cowboy/src/cowboy_spdy.erl
Compiling /deps/cowboy/src/cowboy_router.erl
/home/admin/Repositories/n2o/samples/deps/cowboy/src/cowboy_router.erl:26: Warning: behaviour cowboy_middleware undefined
Compiling /deps/cowboy/src/cowboy_rest.erl
Compiling /deps/cowboy/src/cowboy_req.erl
Compiling /deps/cowboy/src/cowboy_protocol.erl
Compiling /deps/cowboy/src/cowboy_middleware.erl
Compiling /deps/cowboy/src/cowboy_loop_handler.erl
Compiling /deps/cowboy/src/cowboy_http_handler.erl
Compiling /deps/cowboy/src/cowboy_http.erl
Compiling /deps/cowboy/src/cowboy_handler.erl
Compiling /deps/cowboy/src/cowboy_clock.erl
Compiling /deps/cowboy/src/cowboy.erl
Compiling /deps/cowboy/src/cowboy_sup.erl
Compiling /deps/cowboy/src/cowboy_bstr.erl
Compiling /deps/cowboy/src/cowboy_app.erl
Writing /deps/cowboy/ebin/cowboy.app
Compiling /deps/n2o/src/wf_context.erl
Compiling /deps/n2o/src/wf.erl
Compiling /deps/n2o/src/static_file.erl
Compiling /deps/n2o/src/protocols/n2o_text.erl
Compiling /deps/n2o/src/protocols/n2o_rails.erl
Compiling /deps/n2o/src/protocols/n2o_heart.erl
Compiling /deps/n2o/src/protocols/n2o_client.erl
Compiling /deps/n2o/src/protocols/n2o_binary.erl
Compiling /deps/n2o/src/nitrogen/render/wf_tags.erl
Compiling /deps/n2o/src/nitrogen/render/wf_render_elements.erl
Compiling /deps/n2o/src/nitrogen/render/wf_render_actions.erl
Compiling /deps/n2o/src/nitrogen/render/wf_render.erl
Compiling /deps/n2o/src/nitrogen/render/wf_event.erl
Compiling /deps/n2o/src/nitrogen/n2o_nitrogen.erl
Compiling /deps/n2o/src/nitrogen/elements/element_week.erl
Compiling /deps/n2o/src/nitrogen/elements/element_video.erl
Compiling /deps/n2o/src/nitrogen/elements/element_url.erl
Compiling /deps/n2o/src/nitrogen/elements/element_track.erl
Compiling /deps/n2o/src/nitrogen/elements/element_tr.erl
Compiling /deps/n2o/src/nitrogen/elements/element_time.erl
Compiling /deps/n2o/src/nitrogen/elements/element_th.erl
Compiling /deps/n2o/src/nitrogen/elements/element_textbox.erl
Compiling /deps/n2o/src/nitrogen/elements/element_textarea.erl
Compiling /deps/n2o/src/nitrogen/elements/element_tel.erl
Compiling /deps/n2o/src/nitrogen/elements/element_td.erl
Compiling /deps/n2o/src/nitrogen/elements/element_table.erl
Compiling /deps/n2o/src/nitrogen/elements/element_summary.erl
Compiling /deps/n2o/src/nitrogen/elements/element_submit.erl
Compiling /deps/n2o/src/nitrogen/elements/element_style.erl
Compiling /deps/n2o/src/nitrogen/elements/element_source.erl
Compiling /deps/n2o/src/nitrogen/elements/element_select.erl
Compiling /deps/n2o/src/nitrogen/elements/element_search.erl
Compiling /deps/n2o/src/nitrogen/elements/element_script.erl
Compiling /deps/n2o/src/nitrogen/elements/element_reset.erl
Compiling /deps/n2o/src/nitrogen/elements/element_range.erl
Compiling /deps/n2o/src/nitrogen/elements/element_radiogroup.erl
Compiling /deps/n2o/src/nitrogen/elements/element_radio.erl
Compiling /deps/n2o/src/nitrogen/elements/element_q.erl
Compiling /deps/n2o/src/nitrogen/elements/element_progress.erl
Compiling /deps/n2o/src/nitrogen/elements/element_password.erl
Compiling /deps/n2o/src/nitrogen/elements/element_param.erl
Compiling /deps/n2o/src/nitrogen/elements/element_panel.erl
Compiling /deps/n2o/src/nitrogen/elements/element_output.erl
Compiling /deps/n2o/src/nitrogen/elements/element_object.erl
Compiling /deps/n2o/src/nitrogen/elements/element_number.erl
Compiling /deps/n2o/src/nitrogen/elements/element_month.erl
Compiling /deps/n2o/src/nitrogen/elements/element_meter.erl
Compiling /deps/n2o/src/nitrogen/elements/element_meta_link.erl
Compiling /deps/n2o/src/nitrogen/elements/element_meta_base.erl
Compiling /deps/n2o/src/nitrogen/elements/element_meta.erl
Compiling /deps/n2o/src/nitrogen/elements/element_menu.erl
Compiling /deps/n2o/src/nitrogen/elements/element_map.erl
Compiling /deps/n2o/src/nitrogen/elements/element_literal.erl
Compiling /deps/n2o/src/nitrogen/elements/element_list.erl
Compiling /deps/n2o/src/nitrogen/elements/element_link.erl
Compiling /deps/n2o/src/nitrogen/elements/element_li.erl
Compiling /deps/n2o/src/nitrogen/elements/element_legend.erl
Compiling /deps/n2o/src/nitrogen/elements/element_label.erl
Compiling /deps/n2o/src/nitrogen/elements/element_keygen.erl
Compiling /deps/n2o/src/nitrogen/elements/element_ins.erl
Compiling /deps/n2o/src/nitrogen/elements/element_input_time.erl
Compiling /deps/n2o/src/nitrogen/elements/element_input_image.erl
Compiling /deps/n2o/src/nitrogen/elements/element_input_button.erl
Compiling /deps/n2o/src/nitrogen/elements/element_input.erl
Compiling /deps/n2o/src/nitrogen/elements/element_image.erl
Compiling /deps/n2o/src/nitrogen/elements/element_iframe.erl
Compiling /deps/n2o/src/nitrogen/elements/element_html.erl
Compiling /deps/n2o/src/nitrogen/elements/element_hidden.erl
Compiling /deps/n2o/src/nitrogen/elements/element_form.erl
Compiling /deps/n2o/src/nitrogen/elements/element_file.erl
Compiling /deps/n2o/src/nitrogen/elements/element_fieldset.erl
Compiling /deps/n2o/src/nitrogen/elements/element_embed.erl
Compiling /deps/n2o/src/nitrogen/elements/element_email.erl
Compiling /deps/n2o/src/nitrogen/elements/element_dtl.erl
Compiling /deps/n2o/src/nitrogen/elements/element_dropdown.erl
Compiling /deps/n2o/src/nitrogen/elements/element_details.erl
Compiling /deps/n2o/src/nitrogen/elements/element_del.erl
Compiling /deps/n2o/src/nitrogen/elements/element_datetime_local.erl
Compiling /deps/n2o/src/nitrogen/elements/element_datetime.erl
Compiling /deps/n2o/src/nitrogen/elements/element_date.erl
Compiling /deps/n2o/src/nitrogen/elements/element_command.erl
Compiling /deps/n2o/src/nitrogen/elements/element_color.erl
Compiling /deps/n2o/src/nitrogen/elements/element_colgroup.erl
Compiling /deps/n2o/src/nitrogen/elements/element_col.erl
Compiling /deps/n2o/src/nitrogen/elements/element_checkbox.erl
Compiling /deps/n2o/src/nitrogen/elements/element_canvas.erl
Compiling /deps/n2o/src/nitrogen/elements/element_button.erl
Compiling /deps/n2o/src/nitrogen/elements/element_blockquote.erl
Compiling /deps/n2o/src/nitrogen/elements/element_audio.erl
Compiling /deps/n2o/src/nitrogen/elements/element_area.erl
Compiling /deps/n2o/src/nitrogen/actions/action_wire.erl
Compiling /deps/n2o/src/nitrogen/actions/action_transfer.erl
Compiling /deps/n2o/src/nitrogen/actions/action_jq.erl
Compiling /deps/n2o/src/nitrogen/actions/action_event.erl
Compiling /deps/n2o/src/nitrogen/actions/action_confirm.erl
Compiling /deps/n2o/src/nitrogen/actions/action_async.erl
Compiling /deps/n2o/src/nitrogen/actions/action_api.erl
Compiling /deps/n2o/src/nitrogen/actions/action_alert.erl
Compiling /deps/n2o/src/n2o_sup.erl
Compiling /deps/n2o/src/n2o_app.erl
Compiling /deps/n2o/src/handlers/n2o_session.erl
/home/admin/Repositories/n2o/samples/deps/n2o/src/handlers/n2o_session.erl:6: Warning: record state is unused
Compiling /deps/n2o/src/handlers/n2o_secret.erl
Compiling /deps/n2o/src/handlers/n2o_query.erl
Compiling /deps/n2o/src/handlers/n2o_pickle.erl
Compiling /deps/n2o/src/handlers/n2o_mq.erl
Compiling /deps/n2o/src/handlers/n2o_log.erl
Compiling /deps/n2o/src/handlers/n2o_io.erl
Compiling /deps/n2o/src/handlers/n2o_error.erl
Compiling /deps/n2o/src/handlers/n2o_dynroute.erl
Compiling /deps/n2o/src/handlers/n2o_auth.erl
Compiling /deps/n2o/src/formatters/wf_utils.erl
Compiling /deps/n2o/src/formatters/wf_convert.erl
Compiling /deps/n2o/src/formatters/n2o_mochinum.erl
Compiling /deps/n2o/src/formatters/n2o_json.erl
Compiling /deps/n2o/src/endpoints/n2o_websocket.erl
Compiling /deps/n2o/src/endpoints/n2o_document.erl
Compiling /deps/n2o/src/endpoints/cowboy/n2o_dynalo.erl
Compiling /deps/n2o/src/endpoints/cowboy/n2o_cowboy.erl
Compiling /deps/n2o/src/endpoints/cowboy/n2o_bullet.erl
Compiling /deps/n2o/src/endpoints/cowboy/bullet_handler.erl
Writing /deps/n2o/ebin/n2o.app
==> "/home/admin/Repositories/n2o/samples/apps"
==> "/home/admin/Repositories/n2o/samples/apps/review"
Compiling /apps/review/src/web_sup.erl
Compiling /apps/review/src/web_app.erl
Compiling /apps/review/src/users.erl
Compiling /apps/review/src/routes.erl
Compiling /apps/review/src/login.erl
Compiling /apps/review/src/index.erl
/home/admin/Repositories/n2o/samples/apps/review/src/index.erl:12: Warning: variable 'Code' is unused
Compiling /apps/review/src/config.erl
Writing /apps/review/ebin/review.app
DTL Compiling /apps/review/priv/templates/message.html
:0: Warning: function render_tag/3 is unused
DTL Compiling /apps/review/priv/templates/login.html
:0: Warning: function render_tag/3 is unused
DTL Compiling /apps/review/priv/templates/index.html
:0: Warning: function render_tag/3 is unused
Plan Params: []
Ordered: [kernel,stdlib,fs,crypto,compiler,syntax_tools,mnesia,ranch,gproc,
          kvs,cowlib,erlydtl,cowboy,n2o,review,active,mad,rest,sh]
REPL Params: []
Configuration: [{n2o,[{port,8000},
                      {app,review},
                      {route,routes},
                      {log_modules,config},
                      {log_level,config},
                      {log_backend,n2o_log},
                      {event,pickle}]},
                {kvs,[{dba,store_mnesia},
                      {schema,[kvs_user,kvs_acl,kvs_feed,kvs_subscription]}]}]
Applications: [kernel,stdlib,fs,crypto,compiler,syntax_tools,mnesia,ranch,
               gproc,kvs,cowlib,erlydtl,cowboy,n2o,review,active,mad,rest,sh]
Erlang R16B03-1 (erts-5.10.4) [source] [64-bit] [smp:4:4] [async-threads:10] [hipe] [kernel-poll:false]

Backend port not found: inotifywait
Eshell V5.10.4  (abort with ^G)
1> fdlink execv: No such file or directory

=ERROR REPORT==== 6-May-2015::22:48:37 ===
** Generic server fs_server terminating 
** Last message in was {#Port<0.36961>,{exit_status,1}}
** When Server state == {state,#Port<0.36961>,
                               "/home/admin/Repositories/n2o/samples",
                               inotifywait}
** Reason for termination == 
** {port_exit,1}
fdlink execv: No such file or directory

=ERROR REPORT==== 6-May-2015::22:48:37 ===
** Generic server fs_server terminating 
** Last message in was {#Port<0.37024>,{exit_status,1}}
** When Server state == {state,#Port<0.37024>,
                               "/home/admin/Repositories/n2o/samples",
                               inotifywait}
** Reason for termination == 
** {port_exit,1}
fdlink execv: No such file or directory

=ERROR REPORT==== 6-May-2015::22:48:37 ===
** Generic server fs_server terminating 
** Last message in was {#Port<0.37080>,{exit_status,1}}
** When Server state == {state,#Port<0.37080>,
                               "/home/admin/Repositories/n2o/samples",
                               inotifywait}
** Reason for termination == 
** {port_exit,1}
fdlink execv: No such file or directory

=ERROR REPORT==== 6-May-2015::22:48:37 ===
** Generic server fs_server terminating 
** Last message in was {#Port<0.37135>,{exit_status,1}}
** When Server state == {state,#Port<0.37135>,
                               "/home/admin/Repositories/n2o/samples",
                               inotifywait}
** Reason for termination == 
** {port_exit,1}
fdlink execv: No such file or directory

=ERROR REPORT==== 6-May-2015::22:48:37 ===
** Generic server fs_server terminating 
** Last message in was {#Port<0.37173>,{exit_status,1}}
** When Server state == {state,#Port<0.37173>,
                               "/home/admin/Repositories/n2o/samples",
                               inotifywait}
** Reason for termination == 
** {port_exit,1}
fdlink execv: No such file or directory

=ERROR REPORT==== 6-May-2015::22:48:37 ===
** Generic server fs_server terminating 
** Last message in was {#Port<0.37242>,{exit_status,1}}
** When Server state == {state,#Port<0.37242>,
                               "/home/admin/Repositories/n2o/samples",
                               inotifywait}
** Reason for termination == 
** {port_exit,1}

=INFO REPORT==== 6-May-2015::22:48:37 ===
    application: fs
    exited: shutdown
    type: temporary
Apps couldn't be loaded: [fs]


1> 
```

**Comment by 5HT** [comment]
Backend port not found: inotifywait

**Comment by laapsaap** [comment]
Sorry for the last paste, let me try again. I did install inotify tools and tried the again. Will try with a new clone. Thanks

**Comment by 5HT** [comment]
It is not needed just do `./mad rep` after `sudo apt-get install inotify-tools`

**Comment by laapsaap** [comment]
Thanks it works now. You should move `sudo apt-get install inotify-tools` before `./mad deps compile plan repl` part in the README.md.   I didnt pay attention, but it was working the 2nd time I installed inotify-tools.
